crates/spicepod/src/component/runtime.rs

```diff
@@ -56,6 +56,9 @@ pub struct Runtime {
     #[serde(default, skip_serializing_if = "is_default")]
     pub cors: CorsConfig,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub flight: Option<Flight>,
```

Comment:
Alternatively we can make it non-optional to simplify usage, current version is used to align with similar properties implementation, like `auth`